[unable to install 1.0.0-beta.22] npm ERR! code EINVALIDTAGNAME

This is incorrect and trigger an error with NPM:

```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "=>11.0.0" of package "react-relay@=>11.0.0": Tags may not have any characters that encodeURIComponent encodes.
```

I can't install latest version because of this error that has been introduced in https://github.com/zth/rescript-relay/pull/347/files#diff-acc14cfb5379f869ca5516467ad9f5d63a078f5a90f25cd83bd54e395f5957b1R57

I think it should be `>=` no `=>`.

--- a/packages/rescript-relay/package.json
+++ b/packages/rescript-relay/package.json
@@ -54,7 +54,7 @@
   },
   "peerDependencies": {
     "@rescript/react": "*",
-    "react-relay": "=>11.0.0",
+    "react-relay": ">=11.0.0",
     "relay-runtime": "*",
     "rescript": "^9.1.2"
   },

